plugins/ltac2/tac2types.mli

```diff
@@ -84,8 +84,10 @@ type multi = Equality.multi =
 | RepeatStar
 | RepeatPlus
 
+type orientation = bool option
```

Comment:
I don't like this, on the ltac2 side we have `Ltac2 Type orientation := [ LTR | RTL ].` ie orientation does not contain the option

Reply:
Ok, `orientation` is now just an alias for `bool`; however I've kept the `option` in the `rewriting` type (used by the `rewrite` tactic).

Reply:
I forgot to change the Ltac2 side... this is now fixed:

* `Std.setoid_rewrite` in Ltac2 takes an `orientation` parameter rather than `option orientation`; the default is `LTR` when you omit the orientation in the notation;
* the mirror in OCaml `Tac2tactics.setoid_rewrite` also takes an `orientation` (i.e. `bool`) parameter rather than `option orientation`